Core/include/Acts/AmbiguityResolution/ScoreBasedAmbiguityResolution.hpp

```diff
@@ -80,11 +81,17 @@ class ScoreBasedAmbiguityResolution {
     std::size_t nSharedHits = 0;
   };
 
-  /// @brief MeasurementInfo : contains the measurement ID and the detector ID
-  struct MeasurementInfo {
-    std::size_t iMeasurement = 0;
-    std::size_t detectorId = 0;
-    bool isOutlier = false;
+  enum class TrackStateTypes : int {
+    // A measurement not yet used in any other track
+    UnsharedHit = 1,
```

Comment:
why you start with 1?

Reply:
do you really need to specify the number?

Reply:
Starting with one is a convention taken from Athena,  I am not sure we need a specific numbers, But I haven't tested without it.

Reply:
I believe we can simply introduce a conversion function in Athena that converts this enum to the corresponding Athena enum. We do the same for the OutputLevel. 

I'd simplify this by not specifying the actual value. The `class` will forbid implicit conversion to/from integers alsready